ui-speedspacechart/src/components/SpeedSpaceChart.tsx

```diff
@@ -180,14 +182,16 @@ const SpeedSpaceChart = ({
         />
       )}
       <TickLayerX width={width} height={dynamicHeight} store={store} />
-      <ReticleLayer
-        width={width}
-        height={dynamicHeight}
-        heightOffset={dynamicHeightOffset}
-        store={store}
-        showDetailsBox={showDetailsBox}
-        setShowDetailsBox={setShowDetailsBox}
-      />
+      {mouseIsOverSettingsPanel && (
+        <ReticleLayer
```

Comment:
Shouldn't this be the opposite ? When the mouse doesn't hover the settings panel, we want to see the reticle I think (so we need to invert the setState : mouseEnter > true and mouseLeave > false)

```suggestion
      {!mouseIsOverSettingsPanel && (
        <ReticleLayer
```

Reply:
if we agree with your previous comment you're right

Reply:
I agree with Paul on this, if I read only this line, I understand that the reticle layer is displayed only if the mouse is over the settings panel.
It's quite confusing

Reply:
sure, so what do I do, do I use his suggestion `mouseHoveringSettingsPanel` even if it's not about css and apply this second suggestion?

Reply:
anyway I don't care, it's done